sdk/keyvault/azure-keyvault-keys/CHANGELOG.md

```diff
@@ -1,9 +1,23 @@
 # Release History
 
-## 4.3.2 (Unreleased)
+## 4.3.2 (2021-2-10)
```

Comment:
This should be 4.4.0b1. New features require a minor version increment, we don't at this time guarantee the new API won't change (it's pending review), and the clients target a preview Key Vault version by default.